mlflow/onnx.py

```diff
@@ -239,6 +242,31 @@ def __init__(self, path, providers=None):
         else:
             providers = ONNX_EXECUTION_PROVIDERS
 
+        sess_options = onnxruntime.SessionOptions()
+        if "options" in model_meta.flavors.get(FLAVOR_NAME).keys():
+            options = model_meta.flavors.get(FLAVOR_NAME)["options"]
+            inter_op_num_threads = options.get("inter_op_num_threads")
+            intra_op_num_threads = options.get("intra_op_num_threads")
+            execution_mode = options.get("execution_mode")
+            graph_optimization_level = options.get("graph_optimization_level")
+            extra_session_config = options.get("extra_session_config")
+            if inter_op_num_threads:
+                sess_options.inter_op_num_threads = inter_op_num_threads
+            if intra_op_num_threads:
+                sess_options.intra_op_num_threads = intra_op_num_threads
+            if execution_mode:
```

Comment:
Why not force `execution_mode` value to be one of "SEQUENTIAL" and "PARALLEL" ? and we should document it.